common/src/main/java/org/conscrypt/HpkeContext.java

```diff
@@ -0,0 +1,79 @@
+package org.conscrypt;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.Provider;
+import java.security.Security;
+
+class HpkeContext {
```

Comment:
As this class won't be initialized by the caller, but it will be initialized by either `HpkeContextRecipient` and `HpkeContextSender`, how about we mark this class as `abstract`?